beetsplug/ftintitle.py

```diff
@@ -208,7 +301,14 @@ def update_metadata(
         if not drop_feat and not contains_feat(item.title, custom_words):
             feat_format = self.config["format"].as_str()
             new_format = feat_format.format(feat_part)
-            new_title = f"{item.title} {new_format}"
+            # Insert before the first bracket containing remix/edit keywords
+            bracket_pos = find_bracket_position(item.title, bracket_keywords)
+            if bracket_pos is not None:
+                title_before = item.title[:bracket_pos].rstrip()
+                title_after = item.title[bracket_pos:]
+                new_title = f"{title_before} {new_format} {title_after}"
+            else:
+                new_title = f"{item.title} {new_format}"
```

Comment:
It's probably a good idea to define a new method to have this entire logic in a single place

```suggestion
            new_title = self.insert_ft_into_title(title, new_format)
```

```py
...

    def insert_ft_into_title(self, title: str, ft_part:  str) -> str:
        """Make sure the ft part is inserted before any brackets in the title."""
        ...
```

You may as well merge this with `find_bracket_position` logic.